DIPs/DIP10xx.md

```diff
@@ -0,0 +1,84 @@
+# Improved static arrays
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | Lucien Perregaux                                                |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | Will be set by the DIP manager (e.g. "Approved" or "Rejected")  |
+
+## Abstract
+
+Allow a `$` into the brackets for arrays declarations.
+
+## Contents
+* [Rationale](#rationale)
+* [Prior Work](#prior-work)
+* [Description](#description)
+* [Reference](#reference)
+* [Copyright & License](#copyright--license)
+* [Reviews](#reviews)
+
+## Rationale
+
+You may often need to change the content of a static array.
+This feature will save you some time, especially for `betterC` users (because
+dynamic arrays are not available), as you won't need to change the array's length anymore.
+
+## Prior Work
+
+The `$` operator, which is well implemented.
```

Comment:
You might want to read https://github.com/dlang/dmd/pull/3615

Reply:
Oh ! nice.